### PR TITLE
[CI][VSTS] Add required profiles for mac.

### PR DESCRIPTION
**Pull request by mandel-macaque**
fixes: https://github.com/xamarin/maccore/issues/2357

### CODE REVIEW

**Review by dalexsoto — APPROVED** [review]

File: tools/devops/automation/templates/packages/build.yml [hunk 1]
```diff
@@ -323,6 +323,20 @@ steps:
     artifactName: package-internal
   continueOnError: true
 
+# funny enough we need these profiles to build the mac tests
+- bash: |
+    set -x
+    set -e
+    rm -f ~/Library/Caches/com.xamarin.provisionator/Provisions/*p12
+    rm -f ~/Library/Caches/com.xamarin.provisionator/Provisions/*mobileprovision
```
Comment (rolfbjarne) [inline tools/devops/automation/templates/packages/build.yml:334]: The `install-qa-provisioning-profiles.sh` script already does this, so these lines shouldn't be necessary.

**Review by rolfbjarne — CHANGES_REQUESTED** [review]

**Review by spouliot — APPROVED** [review]

**Review by rolfbjarne — APPROVED** [review]

**Comment by mandel-macaque** [comment]
The last comments shows the expected result, one single failure and @spouliot is working on it.

**Pull request merged**